ingest_openapi.yaml

```diff
@@ -100,12 +123,65 @@ components:
               - genomic_id
               - access_method
               - samples
+              - index
+    ClinicalDonorRequest:
+      content:
+        'application/json':
+          schema:
+            $ref: "#/components/schemas/DonorWithClinicalDataField"
```

Comment:
I'd like for the requestBody to contain an object that has a property for `donors` that is the array of DonorWithClinicalData objects:

```
schema:
  type: object
  properties:
    donors:
      type: array
      items:
        $ref: "#/components/schemas/DonorWithClinicalDataField"
  required:
    - donors
```

Reply:
(this would also leave room for other properties that could be stored in the file but aren't needed for ingest, like provenance about what version of the template was used in ETL)

Reply:
Added (mostly) in [0dda517](https://github.com/CanDIG/candigv2-ingest/pull/27/commits/0dda51760049870ff1dc2fc47e37473cbe25f463), but I've kept the array part in the DonorWithClinicalData schema itself to make it easier to write the circular references